examples/async-examples.ts

```diff
@@ -0,0 +1,53 @@
+import {RevAiApiClient, RevAiJobOptions} from 'revai-node-sdk';
+
+/* Initialize your client with your revai access token */
+const token = <YOUR-ACCESS-TOKEN>;
+var client = new RevAiApiClient(token);
+
+/* Get account details */
+var account = await client.getAccount();
+console.log(`Account: ${account.email}`);
+console.log(`Balance: ${account.balance_seconds} seconds`);
+
+/* Submit job and receive job object
+ * Media may be submitted from a local file or from a url by using:
+ * client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3");
+ */
+var jobOptions = new RevAiJobOptions(
+	metadata = "This is an example",
+	skip_diarization = true
+	);
+var job = client.submitJobLocalFile("./example.raw");
+
+console.log(`Job Id: ${job.id}`);
+console.log(`Status: ${job.status}`);
+console.log(`Created On: ${job.created_on}`);
+
+/* Check the status of job */
+var jobDetails = await client.getJobDetails(job.id);
+console.log(`Status for job ${job.id} is ${jobDetails.status}`);
+
+/* Get # of jobs started after a certain job id
+ * client.getListOfJobs() will default to returning all jobs
+ */
+var listOfJobs = await client.getListOfJobs(5, job.id);
+
+/* Get transcript as plain text
+ * Transcripts can also be gotten as Object, Text Stream, or Object Stream via
+ * client.getTranscript{Object, TextStream, ObjectStream}() respectively
```

Comment:
This is a bit confusing to read and I think also inaccurate? All the possible retrieval methods are 
`getTranscriptText`, `getTranscriptTextStream`, `getTranscriptObject`, `getTranscriptObjectStream`, `getCaptionsStream`

Reply:
Instead I would just have all the possible methods below but commented out